_includes/docs/user-guide/rule-engine-2-0/ce-external-nodes.md

```diff
@@ -3,6 +3,72 @@ External Nodes used are used to interact with external systems.
 * TOC
 {:toc}
 
+## AWS Lambda Node
+
+<table  style="width:250px;">
+   <thead>
+     <tr>
+	 <td style="text-align: center"><strong><em>Since TB Version 3.7.1</em></strong></td>
+     </tr>
+   </thead>
+</table> 
+
+![image](/images/user-guide/rule-engine-2-0/nodes/external-aws-lambda.png)
+
+Node publish messages to AWS Lambda (Amazon Lambda).
+
+**Configuration**
+
+// todo: add config image
+
+Function configuration
+
+- **Function name** is required to specify which AWS Lambda function will be invoked and executed.
+- **Qualifier** is used to specify a version or alias of the Lambda function to invoke. By default, Lambda invokes the **$LATEST** version, which is the latest version of the function.
```

Comment:
optional parameter to specify a version or alias of the Lambda function. If the qualifier is not specified, the default qualifier "$LATEST" will be used.